sympy/functions/combinatorial/numbers.py

```diff
@@ -8,7 +8,8 @@
 """
 
 from __future__ import print_function, division
-
+from sets import Set
+from math import sqrt
```

Comment:
Are you sure of this ?
Will using SymPy's `sqrt` be better ?

Reply:
Yeah .It will be same.
I removed import math

Reply:
thanks for suggetion